Assets/Scripts/Models/InventoryManager.cs

```diff
@@ -6,100 +6,89 @@
 // file LICENSE, which is part of this source code package, for details.
 // ====================================================
 #endregion
-
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using MoonSharp.Interpreter;
-using UnityEngine;
 
 [MoonSharpUserData]
 public class InventoryManager
 {
-    // This is a list of all "live" inventories.
-    // Later on this will likely be organized by rooms instead
-    // of a single master list. (Or in addition to.)
-    public Dictionary<string, List<Inventory>> inventories;
+    private static readonly string InventoryManagerLogChanel = "InventoryManager";
 
     public InventoryManager()
     {
-        inventories = new Dictionary<string, List<Inventory>>();
+        Inventories = new Dictionary<string, List<Inventory>>();
     }
 
-    public bool PlaceInventory(Tile tile, Inventory inv)
+    public Dictionary<string, List<Inventory>> Inventories { get; private set; }
```

Comment:
OK. We can make an issue and fix it in another PR. Or add a TODO. Or nothing if you want to do it soon. But something if not, so that we don't forget.